CMSIS/RTOS2/FreeRTOS/Source/cmsis_os2.c

```diff
@@ -1281,12 +1281,10 @@ osTimerId_t osTimerNew (osTimerFunc_t func, osTimerType_t type, void *argument,
       }
 
       mem  = -1;
-      name = NULL;
 
       if (attr != NULL) {
-        if (attr->name != NULL) {
-          name = attr->name;
-        }
+        /* No need to check if NULL, NULL is OK */
```

Comment:
Currently this comment comes immediately after a NULL check and is deceiving. Add meaningful comment, something like:
`/* Take the name from attributes */`

Reply:
Done